setup.py

```diff
@@ -187,9 +178,14 @@ def convert_long_description(raw_readme: str):
         return re.sub(r"(\[[^\(\)\[\]]+\]\()((?!http)[^\(\)\[\]]+\))", f"\\1{url}\\2", raw_readme)
 
 
-def get_version(pkg_dir):
-    with open(os.path.join(pkg_dir, "VERSION")) as f:
+def get_version(pkg_dir: Path):
```

Comment:
According to mypy it is recommended to add the return type as well when you use typehinting (even if the return type is None). So either use no type hinting, or full type hinting.